colcon_package_selection/package_selection/dependencies.py

```diff
@@ -81,6 +89,13 @@ def check_parameters(self, args, pkg_names):  # noqa: D102
                     "Package '{name}' specified with --packages-up-to "
                     'was not found'
                     .format_map(locals()))
+        for pattern in (args.packages_up_to_regex or []):
+            if not any(re.match(pattern, pkg_name) for pkg_name in pkg_names):
+                error_messages.append(
+                    "Pattern '{pattern}' specified with "
+                    "--packages-up-to-regex doesn't match "
+                    'any of the package names'
```

Comment:
Nitpick: it would be good to match the wording from other existing regex options like https://github.com/colcon/colcon-package-selection/blob/a63d232562e93d1bab90fb3728b03ac3f43e561c/colcon_package_selection/package_selection/select_skip.py#L59-L60.

Reply:
Done in a72c6261b5460a338a9be16c536330ce6b838d10